covid19-etl/helper/format_helper.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+import re
+
+
+def format_submitter_id(node, args):
+    """
+    Generates "submitter_id" for node with additional identificator values.
+    Resulting "submitter_id" only contains lowercase letters, digits, underscore and dash.
+
+    Args:
+        node (str): node name for "submitter_id"
+        args (dict): additional arguments to add to "submitter_id"
+
+    Returns:
+        str: generated "submitter_id"
+    """
+    submitter_id = "{}".format(node)
+    for v in args.values():
+        submitter_id += "_{}".format(v)
+
+    submitter_id = submitter_id.lower()
+    submitter_id = re.sub("[^a-z0-9-_]+", "-", submitter_id)
+
+    return submitter_id
```

Comment:
```suggestion
    return submitter_id.strip("-")
```

Reply:
Does it mean `submitter_id` shouldn't have dashes or we want it to contain dashes?

Reply:
it's to remove the extra `-` at the beginning or at the end of the string

Reply:
Ok, in the beginning of the string it shouldn't happen (it expects node name). But wouldn't bad to include this. 👍

Reply:
ah right. at the end it did happen when i was first working on the jhu etl